scripts/level_select.gd

```diff
@@ -0,0 +1,33 @@
+extends Control
+
+const LEVEL_BTN = preload("res://assets/level_button.tscn")
+@export_dir var dir_path
+@onready var grid = $MarginContainer/VBoxContainer/GridContainer
+
+func _ready():
+	get_levels(dir_path)
+
+func get_levels(path) ->void:
+	var dir = DirAccess.open(path)
```

Comment:
Klein probleempje. de levels laat die niet zien in de browser. Ik denk dat het komt door deze DirAccess, omdat je op de browser geen directories meer hebt om te kunnen kijken

(ignore the errors, die zijn niet hierdoor)
![image](https://github.com/user-attachments/assets/1dabb47d-8729-43ed-a7b4-9f6bf143bca2)